.github/workflows/cherry-pick.yml

```diff
@@ -31,320 +31,321 @@ env:
-        run: git rm ${{ steps.changelog.outputs.changelogsToBeDeleted }}
-
-      - name: Commit changes for deletion
-        if: steps.changelog.outputs.changelogsToBeDeleted != '' && steps.changelog.outputs.changelogsToBeDeleted != null
-        run: git commit --no-verify -am "Delete changelog files for ${{ needs.prep.outputs.pr }}"
-
-      - name: Push deletion branch up
-        if: steps.changelog.outputs.changelogsToBeDeleted != '' && steps.changelog.outputs.changelogsToBeDeleted != null
-        run: git push origin delete-changelogs/${{ needs.prep.outputs.pr }}
-
-      - name: Create the PR for deletion branch
-        id: deletion-pr
-        if: steps.changelog.outputs.changelogsToBeDeleted != '' && steps.changelog.outputs.changelogsToBeDeleted != null
-        uses: actions/github-script@v6
-        with:
-          script: |
-            const pr = await github.rest.pulls.create({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              title: "Delete changelog files based on PR ${{ needs.prep.outputs.pr }}",
-              head: "delete-changelogs/${{ needs.prep.outputs.pr }}",
-              base: "trunk",
-              body: "Delete changelog files based on PR #${{ needs.prep.outputs.pr }}"
-            })
-
-            console.log( `::set-output name=deletion-pr::${ pr.data.html_url }` )
-
-      - name: Notify Slack on failure
-        if: ${{ failure() && inputs.skipSlackPing != true }}
-        uses: archive/github-actions-slack@v2.0.0
-        with:
-          slack-bot-user-oauth-access-token: ${{ secrets.CODE_FREEZE_BOT_TOKEN }}
-          slack-channel: ${{ inputs.slackChannelOverride || secrets.WOO_RELEASE_SLACK_CHANNEL }}
-          slack-text: |
-            :warning-8c: Code freeze violation. PR(s) created that breaks the Code Freeze for '${{ needs.prep.outputs.release }}' :ice_cube:
-
-            An attempt to cherry pick PR(s) into outgoing release '${{ needs.prep.outputs.release }}' has failed. This could be due to a merge conflict or something else that requires manual attention. Please check: https://github.com/woocommerce/woocommerce/pull/${{ needs.prep.outputs.pr }}
-
-      - name: Notify Slack on success
-        if: ${{ success() && inputs.skipSlackPing != true }}
-        uses: archive/github-actions-slack@v2.0.0
-        with:
-          slack-bot-user-oauth-access-token: ${{ secrets.CODE_FREEZE_BOT_TOKEN }}
-          slack-channel: ${{ inputs.slackChannelOverride || secrets.WOO_RELEASE_SLACK_CHANNEL }}
-          slack-text: |
-            :warning-8c: Code freeze violation. PR(s) created that breaks the Code Freeze for '${{ needs.prep.outputs.release }}' :ice_cube:
-            
-            Release lead please review:
-
-            ${{ steps.cherry-pick-pr.outputs.cherry-pick-pr }}
-            ${{ steps.deletion-pr.outputs.deletion-pr }}
+
+                      core.setOutput( 'changelogsToBeDeleted', changeLogsToBeDeleted )
```

Comment:
I think there is a typo here. Should be `changelogsToBeDeleted` with lower case `l` on `logs` on the value. The variable name is fine.